### ISSUE TITLE
Issue installing app

### ISSUE THREAD

**Issue by k10sj02** [monsta-hd/scripts#2]
Hi. I'm new to Git, Python, and coding so pardon me if I'm asking a novice question. When I try to drag and drop the extension into Chrome I get the error "Attempted to downgrade" at the top of the page.  In addition, what do I use to manipulate the .yml file in order to fill it with evernote credentials and run the final code.

Any help you could provide in layman's terms would be great. Thanks so much for the help!

<img width="1185" alt="screen shot 2018-03-29 at 8 02 56 pm" src="https://user-images.githubusercontent.com/35823259/38120131-3a7f3f44-338c-11e8-8970-c543b1849f8d.png">

**Comment by yell** [comment]
Hi,
- perhaps the directions in README were slighlty imprecise, but you **don't** need to drag and drop the extension into the chrome, you either use the one provided in the root folder (with which I tested the script) OR you download the extension and again put into the root folder. The script will start its own instance of the browser with the extension attached.
- regarding the `config.yml`: instead of `EVERNOTE_USERNAME` you simply type your mail/username associated with Evernote and instead of `EVERNOTE_PASSWORD` type password for your Evernote account. This (configuration) file is used by the script to enter username and password automatically into the its browser instance. It is handy because you don't need to do it multiple times if you run the script multiple times.

Please reopen the issue if you have further questions.
